packages/assistant-stream/src/core/utils/stream/AssistantTransformStream.ts

```
@@ -0,0 +1,76 @@
+  createAssistantStreamController,
+} from "../../modules/assistant-stream";
+
+export interface AssistantTransformStreamController
+  extends AssistantStreamController {
+  // readonly desiredSize: number | null;
+  // error(reason?: any): void;
+  // terminate(): void;
+}
+
+interface AssistantTransformerFlushCallback {
+  (controller: AssistantTransformStreamController): void | PromiseLike<void>;
```

Biome (1.9.4)

[error] Use a function type instead of a call signature.

Types containing only a call signature can be shortened to a function type.
Safe fix: Alias a function type instead of using an interface with a call signature.


(lint/style/useShorthandFunctionType)